### ISSUE TITLE
invisible and invisible_legend missing from tl_form_field.xlf

### ISSUE THREAD

**Issue by fritzmg** [contao/core-bundle#1318]
The DCA of `tl_form_field` uses a legend called `invisible_legend` and a field with a label reference to
```php
$GLOBALS['TL_LANG']['tl_form_field']['invisible']
```
However, `tl_form_field.xlf` is currently missing both language strings (Contao 4.5.2).

**Comment by leofeyer** [comment]
It seems it has been removed accidentally in Contao 4.5. It exists in version 4.4:

https://github.com/contao/core-bundle/blob/a7ad9038dcc24a2554db67dde1e76997b03796ba/src/Resources/contao/languages/en/tl_form_field.xlf#L365-L367

**Comment by leofeyer** [comment]
Fixed in 8e573d5ff279ba8a04875ed7f8daf674c6bd169a.

**Comment by fritzmg** [comment]
Isn't
```php
$GLOBALS['TL_LANG']['tl_form_field']['invisible']
```
also still missing?

**Comment by fritzmg** [comment]
Just to clarify, I am talking about the label that is referenced [here](https://github.com/contao/core-bundle/blob/4.5/src/Resources/contao/dca/tl_form_field.php#L403). There is no translation for that, as already mentioned in this issue.

**Comment by leofeyer** [comment]
Fixed in 6fb56ae7509c18f171ce2532b17fab70c200e0f3.
